pkg/kp/healthmanager.go

```diff
@@ -208,6 +208,9 @@ func processHealthUpdater(
 			if ok {
 				logger.NoFields().Debug("new health status: ", h.Status)
 				localHealth = &h
+				if !healthEquiv(localHealth, remoteHealth) {
```

Comment:
I expect `checksStream` to receive a value once every second. Of course, the first time there is a transition, `localHealth` won't be equivalent to `remoteHealth`. But if it takes longer than 1s to write that into Consul, due to throttling or Consul slowdowns, the message will be printed multiple times.

Would this still accomplish your goals if it were edge-triggered on `localHealth` changing (without caring about the remote value)?